### PR TITLE
Changed the 'build_gsky.sh' and 'gdal-process/main.go'

### PR DESCRIPTION
**Pull request by asivapra**
I have added the installation scripts in a new dir, 'install'. It is mainly to avoid the scripts writing temp files in the same dir where the rest of the files reside.

### CODE REVIEW

File: gdal-process/main.go [hunk 1]
```diff
@@ -114,9 +117,10 @@ func registerGDALDrivers() {
 	if haveHDF5 {
 		C.GDALRegister_HDF5()
 	}
-	if haveJP2OpenJPEG {
```
Comment (edisonguo) [inline gdal-process/main.go]: As discussed, this was the temporal solution to compile GSKY without JPEG for dev purposes. But this code section needs to stay as we need JPEG for the JPEG version of Sentinel 2 in future. Instead, we need to fix the build script to have GDAL built with JPEG support.

File: install/build_deps.sh [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+wget -q http://download.osgeo.org/gdal/${v}/gdal-${v}.tar.gz
+tar -xf gdal-${v}.tar.gz
+cd gdal-${v}
+./configure --with-geos=yes --with-netcdf
+make -j4
+make install
+)
+rm -rf gdal-${v}
+rm -f gdal-${v}.tar.gz
+echo �Finished installing the dependencies. Now, run the 'build_gsky.sh'�
```
Comment (edisonguo) [inline install/build_deps.sh]: Please note that these are not all the dependencies GSKY require. In addition to those, we will need "build essentials" on ubuntu or "development tools" on centos before building those dependencies. Furthermore, we also need to build Postgresql and Postgis if we treat these as dependencies too. If we want a complete automated build script for deployment purposes, we will need to include these additionals.

File: install/build_gsky.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
+set -xeu
```
Comment (edisonguo) [inline install/build_gsky.sh]: We do not need this script as building GSKY binary will be handled by the Makefile.

File: ows.go [hunk 4]
```diff
@@ -1113,4 +1113,4 @@ func main() {
 	http.HandleFunc("/ows/", owsHandler)
 	Info.Printf("GSKY is ready")
 	log.Fatal(http.ListenAndServe(fmt.Sprintf("0.0.0.0:%d", *port), nil))
-}
+}
```
Comment (edisonguo) [inline ows.go]: Please run `go fmt` before you submit a PR.

File: gdal-process/main.go (hunk 1 above)
Comment (asivapra) [reply gdal-process/main.go]: This issue has now been fixed.

**Pull request closed without merging**